api/src/main/resources/workbench-api.yaml

```diff
@@ -4360,6 +4360,11 @@ definitions:
       firstVisit:
         type: integer
         format: int64
+  DiskType:
+    type: string
+    enum:
+      - pd-standard
```

Comment:
Does swagger generate PD-STANDARD or PD_STANDARD? Shall we use underscore?

Reply:
It generates STANDARD(pd-standard). It's the default definition Inherited from Leo API yml file. I think we could keep it this way?